modules/proxy/proxy_util.c

```diff
@@ -2696,8 +2698,9 @@ PROXY_DECLARE(int) ap_proxy_connect_to_backend(apr_socket_t **newsock,
     int loglevel;
 
     while (backend_addr && !connected) {
-        if ((rv = apr_socket_create(newsock, backend_addr->family,
-                                    SOCK_STREAM, 0, r->pool)) != APR_SUCCESS) {
+        if ((rv = apr_socket_create(newsock, backend_addr->family, SOCK_STREAM, 
+                                    conf->sock_proto, 
```

Comment:
```suggestion
        if ((rv = apr_socket_create(newsock, backend_addr->family,
                                    SOCK_STREAM, conf->sock_proto, 
```

Probably best not to modify the first line.